glustercli/cmd/volume-create.go

```diff
@@ -133,17 +133,16 @@ func volumeCreateCmdRun(cmd *cobra.Command, args []string) {
 		return
 	}
 
-	if len(args) < 2 {
+	if len(args) < 1 {
```

Comment:
what happens when name is specified but bricks are omitted?

Reply:
Converted name as flag(`--name=<name>`). args will contain bricks list only.

Reply:
thanks!